### DISCUSSION TITLE
Backup/Restore credentials

### DISCUSSION THREAD

**Discussion by csphoenix1** [Eugeny/tabby#5571] [Q&A]
Hello,

How can I backup/restore my credentials (SSH password)?

And, where are they stored?

Thank you ! :)

**Comment by Eugeny** [comment]
They're stored in the OS keychain if you didn't enable Vault, and in the Vault, if you did. 
If you're using Vault, you can backup the credentials by backing up your complete config file.

**Reply by Creveoolus** [reply]
What about windows? Where are the passwords stored on windows?
